app/models/datastreams/logo_datastream.rb

```diff
@@ -0,0 +1,3 @@
+class LogoDatastream < ActiveFedora::File
+  include Sufia::FileContent::Versions
```

Comment:
We may need to have some validations in the model to validate file format, size etc?